resources/views/nac.blade.php

```diff
@@ -0,0 +1,132 @@
+@extends('layouts.librenmsv1')
+
+@section('title', __('NAC'))
+
+@section('content')
+<div class="container-fluid">
+    <div class="row">
+    <div class="col-md-12">
+
+    <x-panel title="{{ __('NAC') }}" id="nac-panel">
+        <div class="table-responsive">
+            <table id="nac-grid" data-toggle="bootgrid" class="table table-hover table-condensed table-striped">
+                <thead>
+                <tr>
+                    <th data-column-id="device_id" data-order="asc" data-width="140px">{{ __('Devices') }}</th>
+                    <th data-column-id="port_id" data-width="100px">{{ __('Ports') }}</th>
+                    <th data-column-id="mac_address" data-formatter="tooltip" data-width="170px">{{ __('Mac Address') }}</th>
+                    <th data-column-id="mac_oui" data-formatter="tooltip" data-sortable="false" data-width="130px">{{ __('Vendor') }}</th>
+                    <th data-column-id="ip_address" data-formatter="tooltip" data-width="140px">{{ __('IP Address') }}</th>
+                    <th data-column-id="vlan" data-formatter="tooltip" data-width="60px">{{ __('Vlan') }}</th>
+                    <th data-column-id="domain" data-formatter="nac_domain" data-formatter="tooltip">{{ __('Domain') }}</th>
+                    <th data-column-id="host_mode" data-formatter="nac_mode">{{ __('Host Mode') }}</th>
+                    <th data-column-id="username" data-formatter="tooltip" data-width="250px">{{ __('Username') }}</th>
+                    <th data-column-id="authz_by" data-formatter="tooltip">{{ __('Auth By') }}</th>
+                    <th data-column-id="timeout" data-formatter="time_interval">{{ __('Timeout') }}</th>
+                    <th data-column-id="time_elapsed" data-formatter="time_interval" >{{ __('Time Elapsed') }}</th>
+                    <th data-column-id="time_left" data-formatter="time_interval" data-visible="false">{{ __('Time Left') }}</th>
+                    <th data-column-id="authc_status" data-formatter="nac_authc" data-formatter="tooltip">{{ __('NAC Authc') }}</th>
+                    <th data-column-id="authz_status" data-formatter="nac_authz">{{ __('NAC Authz') }}</th>
+                    <th data-column-id="method" data-formatter="nac_method">{{ __('NAC Method') }}</th>
+
+                </tr>
+                </thead>
+            </table>
+        </div>
+    </x-panel>
+
+    </div>
+    </div>
+</div>
+@endsection
+
+@section('css')
+    <style>
+    </style>
+@endsection
+
+@section('javascript')
+    <script>
+        $(document).ready(function () {
+            locations_grid = $("#nac-grid").bootgrid({
+                ajax: true,
+                rowCount: [25, 50, 100, -1],
+                url: "{{ url('/ajax/table/port-nac') }}",
```

Comment:
Try to use the route() helper over url() when you can.
Add `->name('table.nac')` to the web.php route file and change this to `route('table.nac')`

Reply:
should be ok as well